src/BenchmarkDotNet/Toolchains/Mono/MonoAotToolchain.cs

```diff
@@ -15,7 +15,7 @@ public class MonoAotToolchain : Toolchain
         public static readonly IToolchain Instance = new MonoAotToolchain();
 
         [PublicAPI]
-        public MonoAotToolchain() : base("MonoAot", new Generator(), new MonoAotBuilder(), new Executor(), null)
+        public MonoAotToolchain() : base("MonoAot", new Generator(), new MonoAotBuilder(), new Executor(), new DotNetSdkProvider())
```

Comment:
It appears that MonoAotToolchain uses a variant of Roslyn. So it doesn't need the ISdkProvider (unless Roslyn toolchain needs to validate the sdk also, but it seems different than csproj toolchains).

Reply:
ah ok. Good info. I will definitely need a different approach in implementing this validation to just certain toolchains then.